media-fonts/caligula/caligula-1.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A script font"
```

Comment:
Is this supposed to be in here?

Reply:
I moved that to a different branch and tried to reset the branch.  Please revert.